src/actions/loginActions.js

```diff
@@ -0,0 +1,20 @@
+import {
+  LOGIN_REQUEST,
+  LOGIN_SUCCESS,
+  LOGIN_FAILURE,
+} from '../actionTypes/loginActionType';
+
+export const loginRequestAction = payload => ({
+  type: LOGIN_REQUEST,
+  payload,
+});
+
+export const loginSuccessAction = response => ({
+  type: LOGIN_SUCCESS,
+  token: response.token,
```

Comment:
to be consistent our actions should return an object with keys `type` and `payload`

Reply:
Okay. This is noted.